src/components/NewSnippet.js

```diff
@@ -48,7 +55,29 @@ const Input = styled.input`
   width: 470px;
 `;
 
+const Message = styled.p`
+  font-size: 13px;
+  margin-top: 5px;
+  color: #EE0004;
```

Comment:
css 프로퍼티 순서가 다음과 같이 바뀌어야 할 것 같습니다!
```suggestion
  margin-top: 5px;
  color: #EE0004;
  font-size: 13px;
```